packages/sdk/index.js

```diff
@@ -90,18 +90,23 @@ export class SDK extends EventEmitter {
   /**
    * Creates a Slashtag by name.
    * @param {string} [name] utf8 encoded string
+   * @param {object} [opts] slashtag instantiation options
+   * @param {Corestore} [opts.corestore] corestore
+   * @param {DHT} [opts.dht] dht
```

Comment:
Why do you need to pass a custom DHT too? If you are going to use a custom corestore + custom DHT what do you need the SDK for? Just to generate drive keys?

If so then, it might be a better idea to separate these key generation functionality into their own methods, so you can access them and do with them what you like.

For example:

```js
const { auth, encryptionKey } slashtag.drivestore.generateOptions(feedID)

const drive = new Hyperdrive({ auth, encryptionKey })
```

Reply:
I am simply allowing use of SDK to have same amount of customization as with the direct use of Slashtag.